integration-tests/shard-operator-integration-tests/shard-operator-integration-tests-v2/pom.xml

```diff
@@ -15,4 +15,140 @@
   <version>0.0.1-SNAPSHOT</version>
   <name>SmartEvents :: Shard Operator Integration Tests v2</name>
 
+  <properties>
+    <parallel>false</parallel>
+    <parallel.concurrency.number>3</parallel.concurrency.number>
+
+    <sonar.exclusion>**/RunCucumberTest.java</sonar.exclusion>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.redhat.service.smartevents</groupId>
+      <artifactId>shard-operator-v2</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
You can skip the version declaration here.

Reply:
Thank you. Declaration is skipped.